@eclipseo Build error:
```
/home/runner/work/libopenshot/libopenshot/src/FFmpegWriter.cpp: In member function ‘AVStream* openshot::FFmpegWriter::add_audio_stream()’:
/home/runner/work/libopenshot/libopenshot/src/FFmpegWriter.cpp:1125:10: error: conflicting declaration ‘uint64_t channel_layout’
 1125 | uint64_t channel_layout;
      |          ^~~~~~~~~~~~~~
/home/runner/work/libopenshot/libopenshot/src/FFmpegWriter.cpp:1084:24: note: previous declaration as ‘const uint64_t channel_layout’
 1084 |         const uint64_t channel_layout = info.channel_layout;
      |                        ^~~~~~~~~~~~~~
/home/runner/work/libopenshot/libopenshot/src/FFmpegWriter.cpp:1136:20: error: assignment of read-only variable ‘channel_layout’
 1136 |     channel_layout = c->channel_layout;
      |     ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
gmake[2]: *** [src/CMakeFiles/openshot.dir/build.make:398: src/CMakeFiles/openshot.dir/FFmpegWriter.cpp.o] Error 1
gmake[2]: Leaving directory '/home/runner/work/libopenshot/libopenshot/build'
gmake[1]: *** [CMakeFiles/Makefile2:1810: src/CMakeFiles/openshot.dir/all] Error 2
gmake[1]: Leaving directory '/home/runner/work/libopenshot/libopenshot/build'
gmake: *** [Makefile:166: all] Error 2
```

--- a/src/FFmpegWriter.cpp
+++ b/src/FFmpegWriter.cpp
@@ -1061,7 +1061,7 @@ AVStream *FFmpegWriter::add_audio_stream() {
 		// Set sample rate
 		c->sample_rate = info.sample_rate;
 
-
+const uint64_t channel_layout = info.channel_layout;
 #if HAVE_CH_LAYOUT
 	// Set a valid number of channels (or throw error)
 	AVChannelLayout ch_layout;
@@ -1081,7 +1081,6 @@ AVStream *FFmpegWriter::add_audio_stream() {
 		av_channel_layout_copy(&c->ch_layout, &ch_layout);
 #else
 	// Set a valid number of channels (or throw error)
-	const uint64_t channel_layout = info.channel_layout;
 	if (codec->channel_layouts) {
 		int i;
 		for (i = 0; codec->channel_layouts[i] != 0; i++)
@@ -1092,9 +1091,9 @@ AVStream *FFmpegWriter::add_audio_stream() {
 			}
 		if (codec->channel_layouts[i] == 0)
 			throw InvalidChannels("An invalid channel layout was detected (i.e. MONO / STEREO).", path);
-	} else
-		// Set valid channel layout
-		c->channel_layout = channel_layout;
+		} else
+			// Set valid channel layout
+			c->channel_layout = channel_layout;
 #endif
 
 	// Choose a valid sample_fmt
@@ -1122,7 +1121,6 @@ AVStream *FFmpegWriter::add_audio_stream() {
 	AV_COPY_PARAMS_FROM_CONTEXT(st, c);
 
 int nb_channels;
-uint64_t channel_layout;
 const char* nb_channels_label;
 const char* channel_layout_label;
 

--- a/src/FFmpegWriter.cpp
+++ b/src/FFmpegWriter.cpp
@@ -1061,7 +1061,7 @@ AVStream *FFmpegWriter::add_audio_stream() {
 		// Set sample rate
 		c->sample_rate = info.sample_rate;
 
-const uint64_t channel_layout = info.channel_layout;
+auto channel_layout = info.channel_layout;
 #if HAVE_CH_LAYOUT
 	// Set a valid number of channels (or throw error)
 	AVChannelLayout ch_layout;

--- a/src/FFmpegWriter.cpp
+++ b/src/FFmpegWriter.cpp
@@ -1061,7 +1061,7 @@ AVStream *FFmpegWriter::add_audio_stream() {
 		// Set sample rate
 		c->sample_rate = info.sample_rate;
 
-auto channel_layout = info.channel_layout;
+uint64_t channel_layout = info.channel_layout;
 #if HAVE_CH_LAYOUT
 	// Set a valid number of channels (or throw error)
 	AVChannelLayout ch_layout;
@@ -1131,7 +1131,6 @@ const char* channel_layout_label;
     channel_layout_label = "c->ch_layout.u.mask";
 #else
     nb_channels = c->channels;
-    channel_layout = c->channel_layout;
     nb_channels_label = "c->channels";
     channel_layout_label = "c->channel_layout";
 #endif

